src/checksum/arm64/crc32-arm64.c

```diff
@@ -51,18 +51,19 @@
     __asm__("crc32cb %w[c], %w[c], %w[v]" : [c] "+r"(*&crc) : [v] "r"(+value))
 
 /*
- * __wt_checksum_hw --
- *     Return a checksum for a chunk of memory, computed in hardware using 8 byte steps.
+ * __wt_checksum_with_seed_hw --
+ *     Return a checksum for a chunk of memory, computed in hardware using 8 byte steps. Start with
+ *     the given seed.
  */
 static uint32_t
-__wt_checksum_hw(const void *chunk, size_t len)
+__wt_checksum_with_seed_hw(uint32_t seed, const void *chunk, size_t len)
```

Comment:
My preference is to add the new extra input parameter 'seed' at the end. 
__wt_checksum_hw(const void *chunk, size_t len)
t_checksum_with_seed_hw(const void *chunk, size_t len, uint32_t seed)

Reply:
Is there a good reason for adding seed at the end? I have tried to match it with some of our internal hashing functions. They have the first argument as the seed. Example:
Helper functions in src/checksum/power8/vec_crc32.c
Internal routines in src/checksum/zseries/crc32-s390x.c